### PR TITLE
Fixes Improvised Jetpack (+New custom icons)

### LINKED ISSUE #50396: Improvised jetpack doesn't work
Issue reported from  Round ID: [133905](https://scrubby.melonmesa.com/round/133905) (/tg/Station Terry [EU] [100% LAG FREE] [New-Dedi])

Reporting client version: 513.1507

repro steps:
craft an impro jetpack
try to turn it on

### PR DESCRIPTION
**Pull request by LordVollkorn**
## About The Pull Request

This fixes the improvised jetpack, which currently cannot be activated.
Fixed [#50396](https://github.com/tgstation/tgstation/issues/50396)

The code change was very minor and everything seems to work properly now.
Bonus: New custom icons for this item, better representing what it most likely would look like

## Why It's Good For The Game

Fixes a bug, makes the jetpack usable again and adds custom icons for it

## What the icons look like

complete
complete2 (Icon state while its active)

## Changelog
:cl: LordVollkorn
fix: The improvised jetpack has been fixed. Don't forget that it still has to be filled with gas before usage.
imageadd: New custom icons for the improvised jetpack
/:cl:

### CODE REVIEW

**Comment by Pumpkinoe** [comment]
cool :)

**Comment by Fikou** [comment]
dont say "Apparently an issue was filed about this"
do "Fixed #50396"

File: code/game/objects/items/tanks/jetpack.dm [hunk 1]
```diff
@@ -119,16 +119,14 @@
...
 	volume = 20 //normal jetpacks have 70 volume
 	gas_type = null //it starts empty
 	full_speed = FALSE //moves at hardsuit jetpack speeds
 
 /obj/item/tank/jetpack/improvised/allow_thrust(num, mob/living/user)
-	if(!on)
-		return
 	if((num < 0.005 || air_contents.total_moles() < num))
 		turn_off(user)
 		return
```
Comment (LemonInTheDark) [inline code/game/objects/items/tanks/jetpack.dm:132]: The issue with this proc is less that it still has the old check for on, but more that it's a full copypasta of `/obj/item/tank/jetpack/proc/allow_thrust` with one difference to allow for random failure. It should just check for the random failure, and if it doesn't go through, call parent

Comment (LordVollkorn) [reply code/game/objects/items/tanks/jetpack.dm]: Code has been changed accordingly 👍

**Review by Twaticus — APPROVED** [review]

**Pull request merged**